server.js

```diff
@@ -1,34 +1,30 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');

```

Comment:
Define the 'server' variable and create an HTTP server using 'http.createServer()' method before trying to listen on a port and hostname.